openapi-common/src/main/java/io/micronaut/openapi/swagger/core/util/ModelDeserializer.java

```diff
@@ -56,7 +55,17 @@
  */
 public class ModelDeserializer extends JsonDeserializer<Schema> {
 
-    protected boolean openapi31;
+    static Boolean useArbitrarySchema = false;
+
+    static {
+        if (System.getenv(Schema.USE_ARBITRARY_SCHEMA_PROPERTY) != null) {
```

Comment:
why are we depending on environment variables here? Annotation processors should not use env variables this way.

Reply:
That's not our env vars. This code from `swagger-core`